core/src/main/java/com/klaytn/caver/methods/response/IVote.java

```diff
@@ -0,0 +1,393 @@
+        }
+
+        public static java.lang.Boolean getDeferredTxFee(List<IVote> list) {
+            for(IVote vote: list) {
+                if(vote.getKey().equals(REWARD_USE_DEFERRED_TX_FEE.getKey())) {
+                    return (java.lang.Boolean)vote.getValue();
+                }
+            }
+            return null;
+        }
+
+        public static java.lang.Boolean getDeferredTxFee(IVote vote) {
+            return toBooleanValue(vote.getKey(), vote.getValue());
+        }
+
+        public static String getMinimumStake(Map<String, Object> map) {
+            return (String)map.get(REWARD_MINIMUM_STAKE.getKey());
+        }
+
+        public static String getMinimumStake(List<IVote> list) {
+            for(IVote vote: list) {
+                if(vote.getKey().equals(REWARD_MINIMUM_STAKE.getKey())) {
+                    return (String)vote.getValue();
+                }
+            }
+            return null;
+        }
+
+        public static String getMinimumStake(IVote vote) {
+            return toStringValue(vote.getKey(), vote.getValue());
+        }
+
+        public static BigInteger getProposerUpdateInterval(Map<String, Object> map) {
+            return toBigIntegerValue(map.get(REWARD_PROPOSER_UPDATE_INTERVAL.getKey()));
+        }
+
+        public static BigInteger getProposerUpdateInterval(List<IVote> list) {
+            for(IVote vote: list) {
+                if(vote.getKey().equals(REWARD_PROPOSER_UPDATE_INTERVAL.getKey())) {
+                    return toBigIntegerValue(vote.getValue());
+                }
+            }
+            return null;
+        }
+
+        public static BigInteger getProposerUpdateInterval(IVote vote) {
+            return toBigIntegerValue(vote.getKey(), vote.getValue());
+        }
+
+        public static BigInteger getStakingUpdateInterval(Map<String, Object> map) {
+            return toBigIntegerValue(map.get(REWARD_STAKING_UPDATE_INTERVAL.getKey()));
+        }
+
+        public static BigInteger getStakingUpdateInterval(List<IVote> list) {
+            for(IVote vote: list) {
+                if(vote.getKey().equals(REWARD_STAKING_UPDATE_INTERVAL.getKey())) {
+                    return toBigIntegerValue(vote.getValue());
+                }
+            }
+            return null;
+        }
+
+        public static BigInteger getStakingUpdateInterval(IVote vote) {
+            return toBigIntegerValue(vote.getKey(), vote.getValue());
+        }
+
+
+        public static String toStringValue(String key, Object value) {
+            VoteItem item = fromString(key);
+            if(item == null) {
+                throw new IllegalArgumentException("The " + key + "cannot existed in Vote item");
+            }
+
+            if(!item.getType().equals("String")) {
+                throw new IllegalArgumentException("The type " + key + " cannot convert a string type. It can convert a value to" + item.getType());
+            }
```

Comment:
```suggestion
            VoteItem item = fromString(key);
            if(item == null) {
                throw new IllegalArgumentException("The " + key + "cannot be existed in Vote item");
            }

            if(!item.getType().equals("String")) {
                throw new IllegalArgumentException("The type " + key + " cannot be converted to the string type. It can be converted to " + item.getType());
            }
```
Can we implement `private static Boolean validateKeyValue(String key, String type)`? Then `toXXX` functions can use `validateKeyValues(key, "String")` like this.